modules/bibdk_provider/BibdkUser.php

```diff
@@ -309,7 +309,12 @@ public function deleteSetting($username, $settingtype) {
   /*   * **************  FAVOURITES ************** */
 
   public function setFavourite($username, $agencyid) {
-    $params = array('oui:userId' => $username, 'oui:agencyId' => $agencyid);
+    // Always force encryption on save.
+    $params = array(
+      'oui:userId' => $username,
+      'oui:agencyId' => $agencyid,
+      'oui:encrypted' => 1,
```

Comment:
Her bruger du ikke TRUE/FALSE som i line 345?

Jeg ved godt at det er to forskellige kald, men er der nogen grund til at du ikke bruger boolean?